examples/pxScene2d/src/v8_modules/querystring.js

```diff
@@ -107,6 +107,15 @@ function qsUnescape(s, decodeSpaces) {
 }
 QueryString.unescape = qsUnescape;
 
+/**
+ * unescape from Buffer, code from here
+ * https://github.com/nodejs/node/issues/10824
+ * @param {*} buf the js Buffer
+ */
+function unescapeBuffer (buf) {
+  return buf.toString('hex').replace(/../g, e => `%${e}`);
+}
+QueryString.unescapeBuffer = unescapeBuffer;
```

Comment:
Please remove this. unescapeBuffer had been added in #1573 . Don't overwrite it.